scripts/datacommons/place_cohorts/README.md

```diff
@@ -0,0 +1,59 @@
+# Generating a CohortSet
+
+[generate_cohort_set.py](generate_cohort_set.py) will generate a
+[`CohortSet`](https://datacommons.org/browser/CohortSet)
+node and declare each [`Place`](https://datacommons.org/browser/Place)
+in the given CSV as a [`member`](https://datacommons.org/browser/member).
+
+Notes:
+1. [generate_cohort_set.py](generate_cohort_set.py) includes more
+than the minimal logic, in order to support geo resolution for all geo
+identifiers (Data Commons is moving away from dcid-based references for
+geos).
+1. The names in the CSV are not used. They are included for contributor
+convenience.
+1. Once Data Commons stops accepting place type declarations from arbitrary
+imports, this script should just hardcode "Place" as the place type.
```

Comment:
I don't follow this. We can *always* just set Place as the place type during resolution.

Reply:
This gets written as MCF though, so it's not just for resolution (gets written to graph). It would just add "Place" as a typeOf, which while not so bad, is not preferred?

Reply:
Yes, I understand. We'll drop Place as a type in the presence of alternate types (look at MergeSingleNodeGraphs).

We want MCFs that resolve via otherID to always use Place type.

Reply:
I see it. Nice, I removed this!